packages/ui-kit/src/components/SimpleFilter/SimpleFilter.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { SyntheticEvent, useEffect, useRef } from 'react'
+import { useTopValues } from '../../hooks'
+import { FilterInput, FilterOperator, getTimeZone } from '../../helpers'
+
+import { Autocomplete } from '../Autocomplete'
+import { AutocompleteOption } from '../Autocomplete/Autocomplete.types'
+import { useFilters } from '../FilterProvider/useFilters'
+
+import { SimpleFilterProps } from './SimpleFilter.types'
+import { withContainer } from '../withContainer'
+import { ErrorFallback } from '../ErrorFallback'
+import { Loader } from '../Loader'
+import componentStyles from './SimpleFilter.module.scss'
+import { useLog } from '../Log'
+
+const SimpleFilterComponent = ({
+  autocompleteProps,
+  columnName: columnNameProp = '',
+  query,
+  error,
+  loading,
+  loaderProps,
+  options = []
+}: SimpleFilterProps) => {
+  const id = useRef(Symbol()).current
+
+  const isStatic = !query
+
+  const { filters, setFilters } = useFilters()
```

Comment:
is that posible these states can be type explicit?

Reply:
What do you mean by explicit? I think it's already explicit.

This is the type I'm using for those

```ts
export interface FilterContextValue {
  /** Filters that will be provided to the child components */
  filters: FilterInputWithId[]
  /** Setter function for the Filters */
  setFilters: React.Dispatch<React.SetStateAction<FilterInputWithId[]>>
}
```